Content.Client/Administration/UI/Tabs/ServerTab.xaml

```diff
@@ -9,5 +9,6 @@
         <cc:CommandButton Command="shutdown" Text="{Loc server-shutdown}" />
         <cc:CommandButton Name="SetOocButton" Command="setooc" Text="{Loc server-ooc-toggle}" ToggleMode="True" />
         <cc:CommandButton Name="SetLoocButton" Command="setlooc" Text="{Loc server-looc-toggle}" ToggleMode="True" />
+        <cc:CommandButton Name="SetPanicbunkerButton" Command="panicbunker" Text="{Loc server-panicbunker-toggle}" ToggleMode="True" />
```

Comment:
This being enabled or not should be based on whether they have the perms to change it.

Reply:
It's in `Server` tab so there all commands are `AdminFlags.Server`

Reply:
commandbuttons already do this, this is fine:
![image](https://user-images.githubusercontent.com/19853115/213735129-1e37c798-89fe-4ece-b72c-28fc5c603993.png)